src/metabase/analytics/snowplow.clj

```diff
@@ -32,7 +32,7 @@
   {::account              "1-0-1"
    ::browse_data          "1-0-0"
    ::invite               "1-0-1"
-   ::csvupload            "1-0-0"
+   ::csvupload            "1-0-1"
```

Comment:
According to the process doc, we wait until the commit is merged and only then engage our Snowcat Cloud partners to update it. 

So AFAIK this branch is g2g